lib/rpmte.cc

```diff
@@ -227,6 +236,10 @@ static int addTE(rpmte p, Header h, fnpyKey key, rpmRelocation * relocs)
     rc = 0;
 
 exit:
+    if (rc) {
+	rpmlog(RPMLOG_ERR, _("misformed package %s\n"),
+	       p->NEVRA ? p->NEVRA : (p->name ? p->name : "-"));
+    }
```

Comment:
Uff, this seems to ring a distant bell: the reason we don't have any error messages in here is that we're not guaranteed to have a meaningful handle to refer to, the only one who does is the caller of rpmtsAddInstallElement().

Reply:
So ... I think we're better off just not logging this at all. The caller gets an error code 1, if they don't report it than that's their problem, not ours.

Reply:
And... it's gone.